### PR TITLE
Update a note indicating current state of Federation V1

### PR DESCRIPTION
**Pull request by irfanurrehman**
cc @kubernetes/sig-multicluster-misc 
@quinton-hoole @csbell 

Allow edits from maintainers checkbox

### CODE REVIEW

**Comment by quinton-hoole** [comment]
LGTM

File: _includes/federation-current-state.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+**Note:** The current kubernetes federation API which reuses the kubernetes API 
+resources 'as is' (`Federation V1` as we like to call it), is currently considered 
+alpha for many of its features and there is no clear path to evolve the API to GA. 
+There however is effort ongoing to implement a dedicated federation API, different 
+from k8s API (`Federation V2` as we like to call it); the details of which can be 
+found at [sig-multicluster community page](https://github.com/kubernetes/community/tree/master/sig-multicluster).
+
+----
```
Comment (chenopis) [inline _includes/federation-current-state.md]: If `----` is for a horizontal rule, it's not showing up in the [preview](https://deploy-preview-8165--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-federation/cluster/). There needs to be a break after the paragraph, so just replace this w/ a `<br>`.

**Pull request merged**